static/js/compose.js

```diff
@@ -664,6 +664,7 @@ export function initialize() {
             $("#compose .preview_content"),
             content,
         );
+        resize.reset_compose_textarea_max_height();
```

Comment:
This needs to be called when we end the preview too, right?

Reply:
Also, I guess, should this be in `render_and_show_preview` instead, in case that gets called from other code paths?  I guess the answer is no, since that function is shared with message edit?

Reply:
> This needs to be called when we end the preview too, right?

No, since the compose-textarea height cannot change in preview mode since user cannot type anything or adjust the box size manually.

Reply:
Yes, but don't we need to undo this the change in height that happened when entering preview mode?

Reply:
Good question. But, we are changing height of `preview_message_area` here. We don't need to change its height after it is hidden. When `preview_message_area` is displayed again, we call `reset_compose_textarea_max_height` to adjust its size again. We do need to rename this function now, I guess.